8/7 (#216)
dolthub/weekly-updates: emails/2026-08-07.md

This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 

> It's a big week here at DoltHub. The company turned 8 and we have what some are calling our biggest launch to date.

Reviewer: ```suggestion
It's a big week here at DoltHub. The company turned 8, and we have what some are calling our biggest launch to date.
```

### Doltgres 1.0

> Since we launched Dolt back in 2019, potential customer's biggest ask was "I wish this was Postgres". We listened.

Reviewer 2: ```suggestion
Since we launched Dolt back in 2019, potential customers' biggest ask was "I wish this was Postgres". We listened.
```

> Since we launched Dolt back in 2019, potential customer's biggest ask was "I wish this was Postgres". We listened.

Reviewer: ```suggestion
Since we launched Dolt back in 2019, potential customers' biggest request was "I wish this was Postgres". We listened.
```

> We started on [Doltgres](https://github.com/dolthub/doltgresql), the Postgres-compatible version of Dolt, almost three years ago. Doltgres is finally good enough [to call 1.0: production-ready](https://www.dolthub.com/blog/2026-08-06-doltgres-1-0/). 

Reviewer: ```suggestion
We started on [Doltgres](https://github.com/dolthub/doltgresql), the Postgres-compatible version of Dolt, almost three years ago. Doltgres is finally good enough [to call 1.0 production-ready](https://www.dolthub.com/blog/2026-08-06-doltgres-1-0/). 
```
